### PR TITLE
Parse streetnetwork

### PR DESCRIPTION
**Pull request by mwiktorin**
Not sure if we need gradle test in travis script. It would be nice to have a test report somewhere, autogenerated by travis. Locally i have one in build/reports/tests/test/index.html. I dont know how to get this here

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,4 +27,5 @@ cache:
     - "$HOME/.gradle/caches"
 script:
   - "gradle sonarqube"
+  - "gradle test"
```
Comment (element54) [inline .travis.yml]: This is not needed. You can actually see the test results in sonar

File: src/main/java/org/awesomebakery/model/StreetLink.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+	public static StreetLink fromJson(JsonNode node) {
+		StreetLink link = new StreetLink(node.get("guid").asText());
+		link.source = node.get("source").asText();
+		link.target = node.get("target").asText();
+		link.distance = node.get("dist").asDouble();
+		return link;
+	}
+
+	public static long getSerialversionuid() {
```
Comment (element54) [inline src/main/java/org/awesomebakery/model/StreetLink.java]: Needed?

File: src/main/java/org/awesomebakery/model/StreetNode.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	public static StreetNode fromJson(JsonNode node) {
+		StreetNode streetNode = new StreetNode(node.get("guid").asText());
+		streetNode.name = node.get("name").asText();
+		streetNode.company = node.get("company").asText();
+		streetNode.location = Location.fromJson(node.get("location"));
+		streetNode.type = node.get("type").asText();
+		return streetNode;
+	}
+
+	public static long getSerialversionuid() {
```
Comment (element54) [inline src/main/java/org/awesomebakery/model/StreetNode.java]: Needed?

File: src/main/java/org/awesomebakery/model/StreetLink.java (hunk 2 above)
Comment (azimme2s) [reply src/main/java/org/awesomebakery/model/StreetLink.java]: Same Question

File: src/main/java/org/awesomebakery/model/StreetNode.java (hunk 3 above)
Comment (azimme2s) [reply src/main/java/org/awesomebakery/model/StreetNode.java]: Same Question

**Comment by moriarty** [comment]
You can pass results to Sonar. Although I’m not sure if this is available on SonarCloud.io

Sorry I’m on my mobile: 
https://docs.sonarqube.org/plugins/servlet/mobile?contentId=1442166#content/view/1442166

**Pull request merged**